qemu/tests/cfg/qemu_option_check.cfg

```diff
@@ -8,8 +8,13 @@
             s390x:
                 device_name = virtio-net-ccw
         - e1000:
+            RHEL:
+                only RHEL.6 RHEL.7
             no ppc64,ppc64le,s390x
             device_name = e1000
+        - e1000e:
+            no ppc64,ppc64le,s390x
```

Comment:
So it works for aarch64 and riscv64?

Reply:
Get the limitation from the case of "multi_nics_hotplug". I know it works on ARM, that's aarch64.  But not sure whether it works on riscv32 and riscv64.

Reply:
> Get the limitation from the case of "multi_nics_hotplug". I know it works on ARM, that's aarch64. But not sure whether it works on riscv32 and riscv64.
Okay, let's just keep it this way. Thanks.